lib/megaLogger.js

```diff
@@ -132,23 +136,21 @@
             'LOG': '#000000'
         },
         'dateFormatter': function(d) {
-            return d.getFullYear() + "-" + d.getMonth() + "-" + d.getDate() + " " + d.getHours() + ":" + d.getMinutes() + ":" + d.getSeconds() + "." + d.getMilliseconds();
+            return d.toISOString();
         },
-        'transport': function() {
-            var level = Array.prototype.slice.call(arguments, 0);
-            var args = level.splice(1);
+        'transport': function(level, args) {
```

Comment:
Here i wasn't sure what was the purpose of dealing with Array's `slice` & `splice` functions when you can just provide the plain named arguments :confused: so, this should be a little less overkill.